README.md

```diff
@@ -1,2 +1,45 @@
-# hello_microverse
-Hello Microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse project
+
+> My first Microverse project.
+
+
+## Built With
+
+- HTML & CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+None
+
+### Setup
+None
+
+### Install
+None
+
+### Usage
+Open file with your web browser.
+
+### Run tests
+None
+
+### Deployment
+None
+
+## Authors
+
+👤 **Roberto A. Baños Alvarez**
+
+- GitHub: [@rbanos-mv](https://github.com/rbanos-mv)
+- Twitter: [@RobertoBanosMV](https://twitter.com/RobertoBanosMV)
+- LinkedIn: [Roberto A Baños Alvarez](https://linkedin.com/in/roberto-a-baños-alvarez-500766234)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
It is great you've implemented most parts of the README to match the template given. It would be nice if you add the **Contributing**, **Support**, and **Acknowledgments** sections to finally match this [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md). Please open the link, check those sections and try to make your README file match that as this is required for all headers to be part of your project's README.